Dockerfile

```diff
@@ -0,0 +1,85 @@
+FROM ubuntu:20.04
+
+ENV TERM="xterm-color"
+ARG USER_ID=999
+ARG GROUP_ID=999
```

Comment:
For the integer ID of service accounts, we need to standardize those IDs.  I am not sure if 999/999 is a commonly used by YOCTO, but to ensure clean hand offs in pipelines that may be handled by different containers, we need to ensure such "pseudo-user" accounts are consistent.

Reply:
999/999 was something arbitrary that I put. Maybe 1000/1000 would be better as is the default ID # for the first non-root user. 
Ultimately this is a place holder and when creating the container the user should be override these values.